shapely/speedups/_speedups.pyx

```diff
@@ -49,11 +50,11 @@ def geos_linestring_from_py(ob, update_geom=None, update_ndim=0):
             n = 2
 
         if type(ob) == LineString:
-            return <unsigned long>GEOSGeom_clone_r(handle, g), n
+            return <uintptr_t>GEOSGeom_clone_r(handle, g), n
         else:
-            cs = GEOSGeom_getCoordSeq_r(handle, g)
+            cs = <GEOSCoordSequence*>GEOSGeom_getCoordSeq_r(handle, g)
```

Comment:
I'm surprised these are needed - this is the type defined in geos.pxi. Without this, are you getting warnings? What warnings in particular?